### ISSUE TITLE
Misclick leads to accidental pickup

### ISSUE THREAD

**Issue by vi** [hyst329/OpenFool#33]
Shall picking up while defending be a separate button, but not any click on the playing field?

Or shall attempt to pick up while there is ability to beat the card require some confirmation window?

**Comment by hyst329** [comment]
@vi Yes, it's probably a good idea. Actually, the main concept of the current UI was to get free of various buttons and switches, but the "Done/Take" button is probably necessary to avoid misclicks (the fix is scheduled to introduce in 0.2.7). 
The confirmation window seems to be an overkill, because it take much more time for the user to find the right button and close the annoying window, and the time is precious (especially in online play, which is planned for 0.3.0)

**Comment by czarkoff** [comment]
You can still go without buttons if you limit pickup clicks to the central area of the screen. Giving further hints like highlighting central area would be nice.

Buttons would be an overkill IMO.
